backend/routers/field_officer.py

```
@@ -256,138 +285,158 @@ def officer_check_out(request: OfficerCheckOutRequest, db: Session = Depends(get
+            status_code=500, detail="Check-out failed. Please try again."
+        )
 
 
-@router.post("/field-officer/visit/{visit_id}/upload-images", response_model=VisitImageUploadResponse)
+@router.post(
+    "/field-officer/visit/{visit_id}/upload-images",
+    response_model=VisitImageUploadResponse,
+)
 async def upload_visit_images(
     visit_id: int,
     images: List[UploadFile] = File(..., description="Visit images"),
```

Ruff (0.15.12)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)